ios/FluentUI/People Picker/PersonaCell.swift

```diff
@@ -29,6 +29,22 @@ open class PersonaCell: TableViewCell {
             avatar.state.image = image
         }
 
+        if let loadImageAsync = persona.loadImageAsync {
+            loadImageAsync { image in
+                guard let image = image else {
+                    return
+                }
+                
+                if Thread.isMainThread {
+                    avatar.state.image = image
```

Comment:
we should still check if "avatar" is still valid for the personal cell on the async thread.

Reply:
Sorry, I'm not sure what you mean by "if avatar is still valid".
Are you suggesting capture `avatar` weakly in the `loadImageAsync `closure and verify that it's still in memory?

Reply:
I think it make sense to capture avatar weakly here. Please let me know if the valid check you mentioned is refer to something else.  :)